### ISSUE TITLE
Character stats

### ISSUE THREAD

**Issue by Aprophys** [Valhalla-Gaming/Tracker#8788]
**Describe the issue you're having**:
Can't see the ammount of the intelect of my char. Even relog dont work.
**Explain how you expect it work**:
All stats should we shown in the character menu
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Links to Wowhead, YouTube, etc**:
http://imgur.com/a/2ghzm

**Comment by Moustachos** [comment]
Looks like a character issue, what is your class / specialization?

Did you try to clear your cache and disable all of your addons?

**Comment by Aprophys** [comment]
Yes i deleted my cache reloged manytimes and still no intellect stats.
My class priest spec disc

**Comment by Aprophys** [comment]
At the moment i installed: DejaCharacterStats it shows u every characterstats what u wonna have to show.
You can see it here:

http://i.imgur.com/iT4wXCp.jpg
